todolist/src/App.jsx

```diff
@@ -5,16 +5,29 @@ import Item from './components/Item'
 
 import './App.css'
 
+if (!process.env.REACT_APP_HUB_URLS) {
+  throw new Error('REACT_APP_HUB_URLS environment variable not specified')
+}
+const HUB_URLS = process.env.REACT_APP_HUB_URLS.split(',')
```

Comment:
split will be `undefined` if it's not a string, maybe add a type check next to the previous check